pod.go

```diff
@@ -1152,6 +1168,44 @@ func togglePausePod(podID string, pause bool) (*Pod, error) {
 	return p, nil
 }
 
+func (p *Pod) handleDevices() error {
+	for _, device := range p.devices {
+		// Handle VFIO devices
+		// TODO Add support for handling devices besides VFIO
```

Comment:
Can you open an issue for this (if there isn't already one) and reference it here?

Reply:
Added a reference to the issue here